ui/app/routes/vault/cluster/access/method/section.js

```diff
@@ -1,22 +1,66 @@
-/* eslint-disable prettier/prettier */
-import { set } from '@ember/object';
 import { inject as service } from '@ember/service';
 import Route from '@ember/routing/route';
-import DS from 'ember-data';
 
 export default Route.extend({
   wizard: service(),
+  pathHelp: service('path-help'),
 
-  model(params) {
-    const { section_name: section } = params;
-    if (section !== 'configuration') {
-      const error = new DS.AdapterError();
-      set(error, 'httpStatus', 404);
-      throw error;
+  modelType(backendType, section) {
+    const MODELS = {
+      'aws-client': 'auth-config/aws/client',
+      'aws-identity-whitelist': 'auth-config/aws/identity-whitelist',
+      'aws-roletag-blacklist': 'auth-config/aws/roletag-blacklist',
```

Comment:
It looks like these 3 won't get loaded - is it worth special-casing aws so we can load all of them? It looks like `authConfigs` on the auth-method model is already a hasMany polymorphic relationship so it should support that.

Reply:
yeah that was my next step